doc/source/ray-core/direct-transport.rst

```diff
@@ -316,6 +317,21 @@ Error handling
    * Any unexpected system bugs
 
 
+Advanced: Registering a new tensor transport
+--------------------------------------------
+
+Ray allows users to register new tensor transports for use in RDT at runtime.
+To implement a new tensor transport, you need to implement the abstract interface :class:`ray.experimental.TensorTransportManager <ray.experimental.TensorTransportManager>` defined in `tensor_transport_manager.py <https://github.com/ray-project/ray/blob/master/python/ray/experimental/gpu_object_manager/tensor_transport_manager.py>`__.
+Then you can give `register_tensor_transport` the transport name, supported devices for the transport, and the class that implements `TensorTransportManager`.
```

Comment:
```suggestion
To implement a new tensor transport, implement the abstract interface :class:`ray.experimental.TensorTransportManager <ray.experimental.TensorTransportManager>` defined in `tensor_transport_manager.py <https://github.com/ray-project/ray/blob/master/python/ray/experimental/gpu_object_manager/tensor_transport_manager.py>`__.
Then call `register_tensor_transport` with the transport name, supported devices for the transport, and the class that implements `TensorTransportManager`.
```

Reply:
Can you also add an API link for register_tensor_transport?

Also should specify who this needs to be called by. Driver, parent task, all involved actors, etc?